rs/nns/governance/src/governance.rs

```diff
@@ -775,14 +775,9 @@ impl Proposal {
                             | NnsFunction::DeployGuestosToAllUnassignedNodes => {
                                 Topic::IcOsVersionDeployment
                             }
-                            NnsFunction::NnsCanisterInstall
-                            | NnsFunction::NnsCanisterUpgrade
+                            NnsFunction::NnsCanisterUpgrade
```

Comment:
Why are we leaving NnsCanisterUpgrade in NetworkCanisterManagement versus moving it to ProtocolCanisterManagement?

Reply:
Because it is replaced by `InstallCode` action (as opposed to `ExecuteNnsFunction`), and after a transition period, `NnsCanisterUpgrade` will no longer be supported. It is inaccurate to map it to `ProtocolCanisterManagement` because the canister might not be a protocol canister.